common/shell/commands/sensor_shell.c

```diff
@@ -127,39 +127,42 @@ static int sensor_access(const struct shell *shell, int sensor_num, enum SENSOR_
 
 		char *check_access =
 			(sensor_access_check(sensor_config[sen_idx].num) == true) ? "O" : "X";
+		char *check_poll = (sensor_config[sen_idx].is_enable_polling == true) ? "O" : "X";
```

Comment:
Do we plan to have these only be `0` and `X`?

If so why not make them `char` instead?

Reply:
Pull request has been updated. Please have a look at new commit , thanks!